examples/context/router.yaml

```diff
@@ -1,8 +1,17 @@
+version: 4.0.0
 server:
   cors:
     origins:
       - "https://studio.apollographql.com/"
+telemetry:
+  opentelemetry:
+    jaeger:
+
+override_subgraph_url:
+
```

Comment:
Should this be part of this configuration?

Reply:
Oops